src/App.js

```diff
@@ -1,13 +1,17 @@
-import React, { Component } from 'react';
+import React, {
+	Component
+} from 'react';
 
 class App extends Component {
 	render() {
-		return (
-			<div>
-				<h1>Barefoot Nomad! change while setting up anything!!</h1>
-			</div>
+		return ( <
+			div >
+			<
+			h1 > Barefoot Nomad!change
+			while setting up anything!! < /h1> < /
+			div >
 		);
 	}
 }
 
-export default App;
+export default App;
```

Comment:
can this be fixed if possible 
and add a new line

Reply:
Done, thanks @blaise82